notebooks/114-simplified-mode/114-simplified-mode.ipynb

```diff
@@ -0,0 +1,367 @@
+    "config = dict({\n",
+    "    \"model\": {\n",
+    "        \"model_name\": \"resnet20\",\n",
+    "        \"model\": str(ir_model_xml),\n",
+    "        \"weights\": str(ir_model_bin)\n",
+    "    },\n",
+    "\n",
+    "    \"engine\": {\n",
+    "        \"type\": \"simplified\",\n",
+    "        \"data_source\": CALIB_DIR\n",
+    "    },\n",
+    "\n",
+    "    \"compression\": {\n",
+    "        \"target_device\": \"ANY\",\n",
+    "        \"algorithms\": [\n",
+    "            {\n",
+    "                \"name\": \"DefaultQuantization\",\n",
+    "                \"params\": {\n",
+    "                    \"preset\": \"performance\",\n",
+    "                    \"stat_subset_size\": 300\n",
+    "                }\n",
+    "            }\n",
+    "        ]\n",
+    "    }\n",
+    "})\n",
+    "\n",
+    "with open(\"config.json\", \"w\") as outfile:\n",
+    "    json.dump(config, outfile)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can optimize the model using this configuration file by calling the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pot --output-dir compressed -c config.json --direct-dump"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compare Performance of the Original and Quantized Models\n",
+    "\n",
+    "Finally, we will measure the inference performance of the FP32 and INT8 models. To do this, we use [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html) - OpenVINO's inference performance measurement tool.\n",
+    "\n",
+    "NOTE: For more accurate performance, we recommended running benchmark_app in a terminal/command prompt after closing other applications. Run benchmark_app -m model.xml -d CPU to benchmark async inference on CPU for one minute. Change CPU to GPU to benchmark on GPU. Run benchmark_app --help to see an overview of all command line options."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimized_model_path = Path('compressed/optimized')\n",
+    "optimized_model_xml = optimized_model_path / 'resnet20.xml'\n",
+    "optimized_model_bin = optimized_model_path / 'resnet20.bin'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Inference FP32 model (IR)\n",
+    "!benchmark_app -m $ir_model_xml -d CPU -api async"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Inference INT8 model (IR)\n",
+    "!benchmark_app -m $optimized_model_xml -d CPU -api async"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Demonstration of the results"
```

Comment:
I would anything about what `Demonstration of the results` is about what users should expect to see.